src/cbc_sdk/connection.py

```diff
@@ -321,15 +321,15 @@ def http_request(self, method, url, **kwargs):
                            original_exception=e)
         else:
             if r.status_code >= 500:
-                raise ServerError(error_code=r.status_code, message=r.text)
+                raise ServerError(error_code=r.status_code, message=r.text, uri=uri)
```

Comment:
Given 500 Internal Server Errors are often very generic and don't provide context. Let's determine a way to appropriately append the uri to the message here

Reply:
May need to consider for all uses of ServerError as I see we use this exception in a few places

Reply:
Could consider modifying at the msg level with the following template instead
```
Received error code {0:d} from API ({s})
```

Reply:
Added the URI to the end of the error message reported by `ServerError`, delimited by angle brackets.